app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt

```diff
@@ -163,10 +155,8 @@ class LoginFragment : Fragment() {
 
         rootView.tick.setOnClickListener {
             rootView.sentEmailLayout.visibility = View.GONE
-            if (thisActivity is AppCompatActivity) {
-                thisActivity.supportActionBar?.show()
-                Utils.navAnimVisible(thisActivity.navigationAuth, requireContext())
-            }
+            setToolbar(activity, getString(R.string.login))
```

Comment:
Use Utils.setToolbar, no extra import needed.

Reply:
But, it is okay :+1:

Reply:
okay

Reply:
done